scripts/manage-isos.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -uxeE
+
+usage()
+{
+    echo "----------------------------------------------------------------------------------------"
+    echo "- This script is for internal use at Vates to transfer and manage ISOs generated automatically by our Jenkins server and to store them on our PXE server and ISO SR."
+    echo "- Usage: $0 <XCP-ng version> <Repository> <PXE Server> <Path on the PXE server> <ISO SR Server> <Path on the ISO SR Server> <Path on the PXE Server for the installer>"
+    echo "- All options are mandatory."
+    echo "- Repository: updates, ci, testing..."
+    echo "- Version of XCP-ng: 8.2 or 8.3"
+    echo "- Example: $0 8.3 myuser@my_pxe /my/path/pxe/ iso_user@srv-sr-iso /my/path/on/sriso/ /my/path/pxe/installers/"
+    echo "----------------------------------------------------------------------------------------"
+}
+
+VERSION="$1"
+REPO="$2"
+PXESERVER="$3"
+PXEPATH="$4"
+ISOSR="$5"
+ISOSRPATH="$6"
+PXEPATHINST="$7"
+
+if [ $VERSION != "8.3" ] && [ $VERSION != "8.2" ]; then
+    echo "Unsupported version. Please choose between 8.2 or 8.3."
+    exit
+fi
+
+if [ -z ${PXEPATH} ]; then
+    echo "PXEPATH is empty. Please provide the path on the pxe server."
+    exit
+fi
+
+if [ -z ${ISOSRPATH} ]; then
+    echo "ISOSRPATH is empty. Please provide the path on the SR ISO."
+    exit
+fi
+
+if [ -z ${PXEPATHINST} ]; then
+    echo "PXEPATHINST is empty. Please provide the path on the PXE for the installers."
+    exit
+fi
+
+#get files names
+ISO_FILE_NAME_NI=$(ls *.iso | grep netinstall)
+ISO_FILE_NAME=$(ls *.iso | grep -v netinstall)
+
+#scp to the pxe server
+echo "We're doing the scp to the pxe server for the two isos."
+scp *.iso ${PXESERVER}:${PXEPATH}
+echo "Creating or changing the link latest on the pxe."
+ssh ${PXESERVER} 'ln -sf '${PXEPATH}${ISO_FILE_NAME}' '${PXEPATH}'xcp-ng-'${VERSION}'-'${REPO}'-latest'
```

Comment:
A single set of double-quotes would make the line more readable (like was done in other places), and would allow to single-quote each variable argument to `ln` to also protect that one against special-chars-in-args.